CONTRIBUTING.rst

```diff
@@ -1,7 +1,10 @@
 Contributing
 =========================
 
-As an open source project, Mesa welcomes contributions of many forms.
+As an open source project, Mesa welcomes contributions of many forms and from beginners to experts. If you are
+curious or just want to see what happening. We post our development session agendas
```

Comment:
Probably should be a comma instead of full stop before "We"?

Reply:
Yikes, good call. I was missing an "is" as well. I also added an Oxford comma and made the Matrix portion a sentence to avoid a run-on.